components/announcement_bar/contact_sales/contact_us.tsx

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {FormattedMessage} from 'react-intl';
+
+import {trackEvent} from 'actions/telemetry_actions';
+import './contact_us.scss';
+
+export interface Props {
+    buttonTextElement?: JSX.Element;
+    eventID?: string;
+}
+
+const ContactUsButton: React.FC<Props> = (props: Props) => {
+    const handleContactUsLinkClick = async (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+        e.preventDefault();
+        trackEvent('admin', props.eventID || 'in_trial_contact_sales');
+        window.open('https://mattermost.com/contact-us/', '_blank');
+    };
+
+    return (
+        <button
+            className='contact-us'
+            onClick={(e) => handleContactUsLinkClick(e)}
+        >
+            {props.buttonTextElement || (
+                <FormattedMessage
+                    id={'admin.license.trialCard.contactSales'}
+                    defaultMessage={'Contact sales'}
```

Comment:
I'd say lets pass this down as a proper and make butttonTextElement required.

Reply:
No, it's a standard text and passing the same param from everywhere doesn't make sense. The default value is here because in 99.9% cases this is the text we want. The ability to override is there as a flexibility as it took only 2 lines of code.